springfield/cms/migrations/0006_whatsnewpage.py

```diff
@@ -1,27 +1,173 @@
-# Generated by Django 5.2.2 on 2025-08-13 20:07
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+# ruff: noqa: E501
+# Generated by Django 5.2.5 on 2025-09-08 03:36
 
 import django.db.models.deletion
-import wagtail.fields
 from django.db import migrations, models
 
+import wagtail.fields
+
 
 class Migration(migrations.Migration):
-
     dependencies = [
-        ('cms', '0005_springfieldimage_description'),
-        ('wagtailcore', '0094_alter_page_locale'),
+        ("cms", "0005_springfieldimage_description"),
+        ("wagtailcore", "0095_groupsitepermission"),
```

Comment:
This update removes `('wagtailcore', '0094_alter_page_locale')` as a dependency. Is that intentional?

Reply:
It doesn't remove the previous migration, it's just updating this migration's dependency to the newer wagtailcore dependency (i.e. wagatil 0095 will get run before our cms 0006)